book-domain-service/app/book.py

```diff
@@ -6,3 +6,16 @@ class Book(db.Model):
 
     def __repr__(self):
         return '<Title %r>' % self.title
+
+    def from_json(title_json):
+        new_title = title_json.get('title')
+        if new_title is None or new_title=='':
```

Comment:
The explicit check here for both empty values is nice for readability but if  you wanted to just check for a "falsy" value you could do:

```
if not new_title:
```

That should cover both of the cases you are testing for.